gatling-http/src/main/scala/io/gatling/http/ahc/HttpEngine.scala

```diff
@@ -73,27 +73,33 @@ class HttpEngine(
 
       import httpComponents._
 
+      val debugEnabled = logger.underlying.isDebugEnabled
+
       if (httpProtocol.enginePart.perUserNameResolution) {
         // eager load
         val _ = defaultDnsNameResolver
       }
 
       httpProtocol.warmUpUrl match {
         case Some(url) =>
+          val connectionTimeout: Int = 1000
```

Comment:
no need for a val here, it won't be reused. please just leave it as it was

Reply:
It was hardcoded before, I introduced a val to output it with the info level message. Do you want me to return the hardcoded `.setRequestTimeout(1000)`? (You suggested to inject `connectionTimeout ` in the next comment)

Reply:
Well, the way you've changed it is still hardcoded (it can't be configured, and I fail to see people tuning a configuration option for that, most users aren't even aware of it and it's a good thing), but just at a different place. So yes, please restore it the way it was.

Reply:
Oh, right, you use it in the log message.